common/src/main/java/io/quarkiverse/tekton/pipelinerun/BuildTestPushPipelineRun.java

```diff
@@ -42,7 +41,8 @@ public static PipelineRun create(String projectName, Pipeline pipeline, Optional
              * : Optional.empty())
              * .ifPresent(workspaceBindings::add);
              */
-            WorkspaceBindings.forEmpty(projectName, workspaceName).ifPresent(workspaceBindings::add);
+
+            // WorkspaceBindings.forEmpty(projectName, workspaceName).ifPresent(workspaceBindings::add);
```

Comment:
If you do that, then no workspaces with emptyDir will appear within the generated PipelineRun !

Reply:
Right, but it's not enough to justify moving `WorkspaceBindings` from cli to common. We'll just duplicte the code instead in this case.

Reply:
So we will investigate this point in a separate PR as we discussed to generate properly at `runtime` the Pipelinerun